### DISCUSSION TITLE
[FEATURE]升级应用默认自定义配置文件

### DISCUSSION THREAD

**Discussion by dididaguai** [1Panel-dev/1Panel#6486] [Ideas]
### 1Panel 版本

v1.10.15-lts

### 请描述您的需求或者改进建议

目前升级应用时，默认可以自己对比 docker-compose.yml 文件，但是多个应用时可能忘记进行对比，升级后由于缺少某个参数导致服务异常。
例如onlyoffice应用商店下载默认配置中没有开启WOPI，如果升级多个应用时可能忘记对比，导致服务异常。
【建议】建议升级时默认保存yml配置，只是修改其中镜像的版本号即可。如果应用升级需要修改默认参数，最好在升级时标红提示下
![image](https://github.com/user-attachments/assets/a1dd2d0a-7193-469d-9274-420dfb6e0ca0)

### 请描述你建议的实现方案

【建议】建议升级时默认保存yml配置，只是修改其中镜像的版本号即可

![image](https://github.com/user-attachments/assets/a1dd2d0a-7193-469d-9274-420dfb6e0ca0)

**Comment by wanghe-fit2cloud** [comment]
这种做法可能存在风险，因为新版本的 compose 文件可能会发生变动，仅修改镜像版本可能导致更新无法完全生效，影响新功能的正常运行。
